developer-extension/src/panel/components/json.tsx

```diff
@@ -256,6 +259,7 @@ function JsonText({
     menuItems = (
       <>
         <CopyMenuItem value={descriptor.value}>Copy value</CopyMenuItem>
+        <CopyMenuItem value={getSearchQuery(descriptor, columnPath)}>Copy search query</CopyMenuItem>
```

Comment:
I created the item `getCopyMenuItemsForPath` for copying search query, separating the two for the ease of understanding, because column.path and path might be too mixed up.